cowamm/kpi/competitors/cow_amm/tvl_all_cow_amms_4340356.sql

```diff
@@ -0,0 +1,82 @@
+-- Computes the balances and their dollar value of all CoW AMM pools at every transaction where it changed
```

Comment:
If this query is fast enough, can we replace `query_4059700` with a simpler one that is calling into this one and simply adds a where clause?

Reply:
It's not very fast: 9 minutes vs <1 minute for `query_4059700`